.gitmodules

```diff
@@ -31,12 +25,18 @@
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
 	url = https://github.com/PX4/ecl.git
-[submodule "cmake/cmake_hexagon"]
-	path = cmake/cmake_hexagon
-	url = https://github.com/ATLFlight/cmake_hexagon
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
 	url = https://github.com/PX4/GpsDrivers.git
 [submodule "src/lib/micro-CDR"]
 	path = src/lib/micro-CDR
 	url = https://github.com/eProsima/micro-CDR.git
+[submodule "platforms/nuttx/src/modules/uavcan/libuavcan"]
```

Comment:
Are those effectively two completely separate boards other than the fact one needs to be grab the other's binary? What's a good example?
If that's the case is it sufficient to orchestrate the two different OS builds from the Makefile or do you see the need for tighter integration?